pytrip/vdx.py

```diff
@@ -204,12 +212,45 @@ def voi_names(self):
         names = [voi.name for voi in self.vois]
         return names
 
-    def add_voi(self, voi):
+    def add_voi(self, voi, added_manually=False):
```

Comment:
What about other name for the flag, like `set_spare_color=False` ?

Reply:
or maybe just `set_color`, because we are not certain about using spare colors - there can be none
`set_color` informs only about the fact that color for added voi will be set - method will be more hermetic

Reply:
good idea

Reply:
Done